doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1238 @@
+
+Cryptography samples
+--------------------
+
+* Added:
+
+    * :ref:`crypto_spake2p` sample.
+    * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board for all crypto samples.
+    * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board in all crypto samples, except :ref:`crypto_persistent_key` and :ref:`crypto_tls`.
+    * Support for the :ref:`zephyr:nrf9151dk_nrf9151` board for all crypto samples.
+    * Support for the :ref:`nRF9161 DK <ug_nrf9161>` board for the :ref:`crypto_test` sample.
+
+Common samples
+--------------
+
+* Added a description about :file:`samples/common` and their purpose in the :ref:`samples` and :ref:`building_advanced` pages (:ref:`common_sample_components`).
+
+Debug samples
+-------------
+
+* :ref:`memfault_sample` sample:
+
+  * Added support for Thingy:91 X.
+
+DECT NR+ samples
+----------------
+
+* Added the :ref:`nrf_modem_dect_phy_hello` sample.
+
+
+Enhanced ShockBurst samples
+---------------------------
+
+* :ref:`esb_prx` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15` boards.
+
+Keys samples
+------------
+
+* Added support for the :ref:`zephyr:nrf9151dk_nrf9151` and the :ref:`zephyr:nrf9161dk_nrf9161` boards for all keys samples.
+
+Matter samples
+--------------
+
+* Added:
+
+  * Test event triggers to all Matter samples and enabled them by default.
+    By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.
+    To get started with using test event triggers in your Matter samples and to understand the capabilities of this feature, refer to the :ref:`ug_matter_test_event_triggers` page.
+
+  * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` with the ``nrf54l15pdk/nrf54l15/cpuapp`` board target to the following Matter samples:
+
+    * :ref:`matter_template_sample` sample.
+    * :ref:`matter_light_bulb_sample` sample.
+    * :ref:`matter_light_switch_sample` sample.
+    * :ref:`matter_thermostat_sample` sample.
+    * :ref:`matter_window_covering_sample` sample.
+
+    DFU support for the nRF54L15 PDK is available only for the ``release`` build type.
+
+  * Support for Matter over Thread on the :ref:`zephyr:nrf54h20dk_nrf54h20` with the ``nrf54h20dk/nrf54h20/cpuapp`` board target to the following Matter samples:
+
+    * :ref:`matter_lock_sample` sample.
+    * :ref:`matter_template_sample` sample.
+
+    DFU, factory data, and PSA Crypto API are not currently supported for the nRF54H20 DK.
+
+* Updated:
+
+  * Matter samples that used :ref:`zephyr:nrf-ieee802154-rpmsg-sample` or :ref:`multiprotocol-rpmsg-sample` radio core firmware, now use the :ref:`ipc_radio`.
+  * Enabled the Bluetooth® LE Extended Announcement feature for all samples, and increased advertising timeout from 15 minutes to 1 hour.
```

Comment:
Bluetooth®- trademark to be moved to the first instance (probably in highlights)